bcipy/acquisition/tests/datastream/test_generator.py

```diff
@@ -96,7 +95,8 @@ def test_file_generator_end(self):
             for _ in range(row_count):
                 next(gen)
 
-            with pytest.raises(StopIteration):
+            with self.assertRaises(RuntimeError):
```

Comment:
Calling `next` on a depleted Generator should raise a `StopError` exception, which is what we're explicitly testing in this situation. A RuntimeError is too general and could be the result of a different unexpected issue.